File: CodeChef/Self Learning/Getting Started/Enormous_Input_Test.cpp
// CodeChef - INTEST - Enormous Input Test
// https://www.codechef.com/LP0TO101/problems/INTEST

#include "bits/stdc++.h"
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int n, k;
    cin >> n >> k;
    int ans = 0;
    for(int i = 1; i <= n;  ++i){
        int t;
        cin >> t;
        ans += t % k == 0;
    }
    cout << ans << endl;
}

File: CodeChef/Self Learning/Getting Started/Enormous_Input_Test.py
# CodeChef - INTEST - Enormous Input Test
# https://www.codechef.com/LP0TO101/problems/INTEST

import sys
input = sys.stdin.readline

n, k = map(int, input().split())
ans = 0
for _ in range(n):
    t = int(input())
    ans += t % k == 0
    
print(ans)